submissions/dikun/js-dom/script.js

```diff
@@ -0,0 +1,101 @@
+            img: 'img/sophia.webp',
+            descr: "Chersonesos was an ancient Greek colony founded approximately 2500 years ago in the southwestern part of Crimea, known then as Taurica. The ancient city is located on the shore of the Black Sea at the outskirts of Sevastopol and has been nicknamed the 'Ukrainian Pompeii'. Chersonesos' ancient ruins are presently located in one of Sevastopol's suburbs. They are today a popular tourist attraction, protected by the state as an archaeological park. The buildings mix influences of Greek, Roman and Byzantine culture. The defensive wall is hundreds of meters long. Buildings include Roman amphitheatre and a Greek temple."
+        },
+        khotyn: {
+            title: 'the khotyn fortress',        
+            img: 'img/khotyn.jpg',
+            descr: "The Khotyn Fortress is a fortification complex located on the shores of the Dniester River in Khotyn in western Ukraine. The construction of the current fortress was started in 1325. The fortress is a large tourist attraction in the area and Ukraine. Thanks to its preserved historical atmosphere the fortress is often used as a setting for movies. It has represented various French and English castles, historical locations, and fortifications. Most recently, it appeared in a Russian movie Taras Bulba, The Three Musketeers, or the Arrows of Robin Hood."
+        }
+    };
+
+    const list = document.querySelector(".list_items");
+    const content = document.querySelector(".content");
+    
+    const createListHtml = (allMyWonders) => {
+         Object.values(allMyWonders).forEach((key) => {
+            let listItem = document.createElement('li');
+            let btn = document.createElement('button');
+            
+            listItem.classList.add("list_item");
+            btn.classList.add("list_btn");
+            btn.innerText = key.title;
+            listItem.append(btn);
+
+            list.append(listItem);            
+        });  
+    };
+   
+    createListHtml(sevenWonders);
+    
+    const createMainContent  = (key) => { 
+        content.innerHTML = '';  
+        const titleContent = document.createElement('h2');
+        titleContent.classList.add("content_title");
+        titleContent.innerText = key.title;           
+        content.append(titleContent);
+        
+        const imgContent = document.createElement('img');
+        imgContent.classList.add("content_img");
+        imgContent.src =  key.img;
+        content.append(imgContent);
+
+        const descrContent = document.createElement('p');
+        descrContent.classList.add("content_descr");
+        descrContent.innerText = key.descr;
+        content.append(descrContent);          
+        
+    };     
+
+   function changeContentByTitle(allMyWonders) {
+    let keysNames = Object.values(allMyWonders);   
+    
+    createMainContent(keysNames[0]);
+
+    function buttonEvent (e) {
+        const btnName = e.target.innerText.toLowerCase(); 
```

Comment:
If I will translate website - it will not work.
And anyone can change this value, basically. Editors, translators, UI/UX designers and your code will be broken.

You should rely only on "dev only" data - some entities which is used only by developers. In your case you can use `data-*`.